src/initialization/InitElement.cpp

```diff
@@ -434,6 +435,23 @@ namespace detail
                     read_element(sub_element_name, m_lattice, nslice_default, mapsteps_default);
                 }
             }
+        } else if (element_type == "linear_map")
+        {
+            auto a = detail::query_alignment(pp_element);
+
+            elements::LinearTransport::Map6x6 transport_map;
+            for (int i=1; i<=6; ++i) {
+                for (int j=1; j<=6; ++j) {
```

Comment:
Memo to Axel: default is F-order of `amrex::Array2`. Double check loop indices are effective.